chapters/handling-data.tex

```diff
@@ -8,19 +8,29 @@
 asked by development researchers grow,
 so does the (rightful) scrutiny under which methods and results are placed.
 This scrutiny involves two major components: data handling and analytical quality.
-Performing at a high standard in both means that research participants
-are appropriately protected,
-and that consumers of research can have confidence in its conclusions.
-
-What we call ethical standards in this chapter is a set of practices for research transparency and data privacy that address these two components.
-Their adoption is an objective measure of to judge a research product's performance in both.
-Without these transparent measures of credibility, reputation is the primary signal for the quality of evidence, and two failures may occur:
+Performing at a high standard in both means that
+consumers of research can have confidence in its conclusions,
+and that research participants are appropriately protected.
+What we call ethical standards in this chapter is a set of practices
+for research quality and data privacy that address these two principles.
+
+Neither quality nor privacy is an ``all-or-nothing'' objective.
+We expect that teams will do as much as they can to make their work
+conform to modern practices of credibility, transparency, and replicability.
+Similarly, we expect that teams will ensure the privacy of participants in research
+by intelligently assessing and proactively averting risks they might face.
+We also expect teams will report what they have and have not done
+in order to provide objective measures of a research product's performance in both.
+Without these transparent measures of quality, reputation is the primary signal for the quality of evidence, and two failures may occur:
 low-quality studies from reputable sources may be used as evidence when in fact they don't warrant it,
 and high-quality studies from sources without an international reputation may be ignored.
 Both these outcomes reduce the quality of evidence overall.
-Even more importantly, they usually mean that credibility in development research accumulates at international institutions and top global universities instead of the people and places directly involved in and affected by it.
-Simple transparency standards mean that it is easier to judge research quality, and making high-quality research identifiable also increases its impact.
-This section provides some basic guidelines and resources for collecting, handling, and using field data ethically and responsibly to publish research findings.
+Even more importantly, they usually mean that credibility in development research accumulates at international institutions
+and top global universities instead of the people and places directly involved in and affected by it.
```

Comment:
I was about to write something in relation to this as well. I think it can be made even more specific.

> Even more importantly, the only way to determine credibility without transparency is to judge research only based on where it is done and by who which concentrates credibility at better-known international institutions and global universities at the expense of the people and organization directly involved in and affected by it.

Reply:
Implemented in a05d9aba184ba851b9f25d48a0ab26d8dc8bcaa2

Reply:
Resolved in my opinion. @luizaandrade ?

Reply:
I think the one this is missing from my original point is that transparency does not guarantee that people and organizations directly involved and affected by research will gain credibility unless they have sound practices.

But maybe I'm just missing the point of this sentence?

Reply:
"at the expense of quality research done by people and organizations directly involved in and affected by it."? In fbec440a9424fdec0f9969c7ab54de777ec717b0 including the typo fix.

Reply:
yep, that's great!